packages/e2e-tests/scenarios/module-metadata/module-metadata.test.ts

```diff
@@ -0,0 +1,21 @@
+import * as path from "path";
```

Comment:
Can we translate this E2E test into a integration test instead.

IIRC the E2E tests were built for querying the Sentry API for uploaded artifacts.

Reply:
I completely missed the fact there are separate integration tests 🙈